.travis.yml

```diff
@@ -11,4 +11,4 @@ script:
   - ./go.test.sh
 
 after_success:
-  - bash <(curl -s https://codecov.io/bash)
+  - include_cov=coverage.txt bash <(curl -s https://codecov.io/bash)
```

Comment:
why not use the `-f` argument which is documented in the output of the `--help` message?

the include_cov looks like a private variable which is not documented.
even in codecov.sh it has no comment attached to it... so no documentation

Reply:
You may be right. It took me a while to find a solution and this was the first one that worked for me: https://github.com/elliotchance/c2go/blob/master/.travis.yml#L47

I will try `-f` later and let you know.